platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -593,6 +593,20 @@ public float getTilt() {
         return nativeGetTilt(mapPointer);
     }
 
+    /**
+     * Run flight animation to change postion and zoom  of the map
+     * @param position LngLat of the position to set
+     * @param zoom Zoom level; lower values show more area
+     * @param duration Time in milliseconds to ease to given zoom
+     * @param speed If duration is 0, speed is used as factor to change the duration that is
+     *              calculated for the duration of the flight path. (Recommended range 0.1 - 2.0)
+     */
+    public void flyTo(@NonNull final LngLat position, final float zoom, final float duration, final float speed) {
```

Comment:
If `duration` is in milliseconds then we should make it an `int` for consistency with the other easing methods.

Reply:
I still think overloads may be useful, but we can always add them later without breakage.

Reply:
Thanks for the suggestions! I'll finish this up on Monday